src/main-oe/env.c

```diff
@@ -0,0 +1,84 @@
+#include "host/env.h"
+#include <errno.h>
+#include <limits.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "shared/util.h"
+
+uint64_t size_str_to_uint64(const char* str, uint64_t def, uint64_t max)
+{
+    uint64_t r;
+    char* endptr;
+    errno = 0;
+    r = (uint64_t)strtoull(str, &endptr, 10);
+    if (r == ULONG_MAX && errno == ERANGE)
+    {
+        r = def;
+    }
+    int m = 1;
+    switch (*endptr)
+    {
+        case 'G':
+        case 'g':
+            m *= 1024;
```

Comment:
Please add `__attribute__((fallthrough));` if fall-through is intended (I believe it is here, but I shouldn't have to read the entire switch statement to know that this is intentional).

Reply:
I just moved this file. Looks intentional though, so I'll add the attributes.